MoneyHoney/redux/reducer.jsx

```diff
@@ -0,0 +1,10 @@
+const initState = {};
```

Comment:
initSate is a function name, pls call the variable `initialSate`

Reply:
renamed